packages/client/src/lib.js

```diff
@@ -26,7 +26,7 @@ import {
   Blockstore
 } from './platform.js'
 
-const MAX_PUT_RETRIES = 5
+const MAX_PUT_RETRIES = 1
```

Comment:
Just 1?

Reply:
reverting, I was using this to test and accidentally put it in